extension/src/pipeline/quickPick.test.ts

```diff
@@ -588,17 +617,109 @@ describe('pickPlotConfiguration', () => {
     mockedQuickPickOne.mockResolvedValue('simple')
     mockedGetInput.mockResolvedValue('simple_plot')
     mockedQuickPickUserOrderedValues
+      .mockResolvedValueOnce([
```

Comment:
This test was actually incorrectly written. See https://github.com/iterative/vscode-dvc/pull/4797/files#r1355869332